Owncloud iOs Client/Tabs/SettingTab/SettingsViewController.m

```diff
@@ -1762,260 +1769,21 @@ -(void)setPropertiesTouchIDToState:(BOOL)isTocuhIDActive{
     }
 }
 
-
-#pragma mark - Instant Upload - Location Support
-
-- (void)switchInstantUploadTo:(BOOL)value {
-     [self.switchInstantUpload setOn:value animated:NO];
-}
-
--(void)setPropertiesInstantUploadToState:(BOOL)stateInstantUpload{
-    
-    AppDelegate *app = (AppDelegate *)[[UIApplication sharedApplication] delegate];
-    if (app.activeUser.username == nil) {
-        app.activeUser = [ManageUsersDB getActiveUser];
-    }
-    
-    if (stateInstantUpload) {
-        [self switchInstantUploadTo:YES];
-        app.activeUser.instantUpload = YES;
-        [ManageAppSettingsDB updateInstantUploadTo:YES];
-    } else {
-        [self switchInstantUploadTo:NO];
-        app.activeUser.instantUpload = NO;
-        [ManageAppSettingsDB updateInstantUploadTo:NO];
-        [[ManageLocation sharedSingleton] stopSignificantChangeUpdates];
-    }
-}
-
--(void)setDateInstantUpload{
```

Comment:
The Instant Upload date check now takes place in the ```attemptUpload:``` method of ```InstantUpload``` in the assets fetch, where the query is filtered by the last upload date:

```
NSDate *lastInstantUploadedAssetCaptureDate = [NSDate dateWithTimeIntervalSince1970:ACTIVE_USER.dateInstantUpload];

PHFetchOptions *newAssetsFetchOptions = [PHFetchOptions new];
newAssetsFetchOptions.sortDescriptors = @[[NSSortDescriptor sortDescriptorWithKey:@"creationDate" ascending:YES]];
newAssetsFetchOptions.predicate = [NSPredicate predicateWithFormat:@"creationDate > %@", lastInstantUploadedAssetCaptureDate];
```